mingw-w64-arpack/PKGBUILD

```
@@ -1,67 +1,152 @@
 # $Id$
-# Maintainer (ArchLinux): Alexander Rødseth <rodseth@gmail.com>
+# Maintainer (ArchLinux): Alexander Rodseth <rodseth@gmail.com>
 # Contributor (ArchLinux): William Rea <sillywilly@gmail.com>
 # Contributor (ArchLinux): Stefan Husmann <stefan-husmann@t-online.de>
 # Maintainer (MSYS2): Ray Donnelly <mingw.android@gmail.com>
 
 _realname=arpack
 pkgbase=mingw-w64-${_realname}
-pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
+         $([[ "${CARCH}" == "i686" ]] || echo "${MINGW_PACKAGE_PREFIX}-${_realname}64"))
 pkgver=3.8.0
-pkgrel=2
+pkgrel=3
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
 pkgdesc="Fortran77 subroutines designed to solve large scale eigenvalue problems (mingw-w64)"
 url='https://forge.scilab.org/index.php/p/arpack-ng/'
 license=('BSD')
-depends=("${MINGW_PACKAGE_PREFIX}-gcc-libgfortran"
+depends=($([[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]] || echo "${MINGW_PACKAGE_PREFIX}-gcc-libgfortran")
          "${MINGW_PACKAGE_PREFIX}-openblas")
-makedepends=("${MINGW_PACKAGE_PREFIX}-gcc-fortran")
-provides=("${MINGW_PACKAGE_PREFIX}-arpack-ng")
+if [ "$CARCH" = "x86_64" ]; then
+	depends+=("${MINGW_PACKAGE_PREFIX}-openblas64")
+fi
+makedepends=($([[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]] || echo "${MINGW_PACKAGE_PREFIX}-gcc-libgfortran")
```

Is this a typo? Did you want to write `gcc-fortran`?
Is it necessary to make-depend on packages that are already in the `depends` listing?

---

Yes, it is a typo, should be "${MINGW_PACKAGE_PREFIX}-gcc-fortran"

---

Would it make sense to remove the condition to let the CI fail early? (See also the other comment.)